glide.lock

```diff
@@ -1,5 +1,5 @@
-hash: 94c34706c54309813f1552d1977f432db42172f974fbeadb269e378c5d1d9c9e
-updated: 2018-06-05T14:47:51.914081842+08:00
+hash: b8371ccd3fdacb75f39f845948278673db65ed4771cb510b49df4cd54c28eb49
```

Comment:
Split the changes to two commits: one for vendor updates and the other one for adding tests.

Reply:
fixed